src/parsers/accounts.js

```diff
@@ -15,7 +15,7 @@ import { getTokenMetadata, isLowerCaseMatch } from '@rainbow-me/utils';
 /**
  * @desc parse account assets
  * @param  {Object} [data]
- * @return {Array}
+ * @return The array of parsed account assets.
```

Comment:
These return type lines actually caused TypeScript compilation to fail, since they are interpreted as `any[]` as opposed to the correct type. Without them, TS infers the more detailed type and compiles. There might be a better way to resolve this though!